NineChronicles.Headless/GraphTypes/StateQuery.cs

```diff
@@ -137,15 +138,56 @@ public StateQuery()
                                     }
                                 }
 #pragma warning disable CS0618 // Type or member is obsolete
-                                arenastate.OrderedArenaInfos.AddRange(arenaInfos.OrderByDescending(a => a.Score).ThenBy(a => a.CombatPoint));
+                                arenastate.OrderedArenaInfos.AddRange(arenaInfos.OrderByDescending(a => a.Score)
+                                    .ThenBy(a => a.CombatPoint));
 #pragma warning restore CS0618 // Type or member is obsolete
                             }
                         }
+
                         return arenastate;
                     }
 
                     return null;
                 });
+            Field<NonNullGraphType<ListGraphType<ArenaInformationType>>>(
+                name: "arenaInformation",
+                description: "List of arena information of requested arena and avatar list",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<IntGraphType>>
+                    {
+                        Name = "championshipId",
+                        Description = "Championship ID to get arena information"
+                    },
+                    new QueryArgument<NonNullGraphType<IntGraphType>>
+                    {
+                        Name = "round",
+                        Description = "Round of championship to get arena information"
+                    },
+                    new QueryArgument<NonNullGraphType<ListGraphType<AddressType>>>
```

Comment:
If you don't want to receive `[null]` as `avatarAddresses`, it should type like this:

```suggestion
                    new QueryArgument<NonNullGraphType<ListGraphType<NonNullGraphType<AddressType>>>>
```